ipa-server-configure-first

```diff
@@ -208,6 +208,14 @@ else
 			fi
 		done
 		cp /etc/volume-version /data/volume-version
+
+		if ! [ -f /etc/ipa/ca.crt ] && [ -f /root/ipa.csr ] ; then
+			cp -p /root/ipa.csr /data/ipa.csr
+			echo "FreeIPA server certificate signing request file is in volume in file ipa.csr."
```

Comment:
Changed as suggested in d5bf4907a3f801b5025fd98e96ea85818fabdb22, rebased the rest on top -> e4fed67cc2feaa20c5a84b69b39e579b29a93c66.